client/driver/driver.go

```diff
@@ -133,7 +133,6 @@ func TaskEnvironmentVariables(ctx *ExecContext, task *structs.Task) environment.
 		if len(task.Resources.Networks) > 0 {
 			network := task.Resources.Networks[0]
 			env.SetTaskIp(network.IP)
```

Comment:
It is applicable to all the drivers tho. If I do an exec and ask for dynamic ports I still need to know which ports to bind to.